community/kernel/src/main/java/org/neo4j/kernel/impl/store/counts/RecountFromTransactions.java

```diff
@@ -1,145 +0,0 @@
-/*
- * Copyright (c) 2002-2015 "Neo Technology,"
- * Network Engine for Objects in Lund AB [http://neotechnology.com]
- *
- * This file is part of Neo4j.
- *
- * Neo4j is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-package org.neo4j.kernel.impl.store.counts;
-
-import java.io.File;
-import java.io.IOException;
-import java.util.List;
-
-import org.neo4j.io.fs.DefaultFileSystemAbstraction;
-import org.neo4j.io.fs.FileSystemAbstraction;
-import org.neo4j.io.pagecache.PageCache;
-import org.neo4j.kernel.impl.api.CommandApplierFacade;
-import org.neo4j.kernel.impl.api.CountsRecordState;
-import org.neo4j.kernel.impl.api.CountsStoreApplier;
-import org.neo4j.kernel.impl.api.CountsVisitor;
-import org.neo4j.kernel.impl.store.StoreFactory;
-import org.neo4j.kernel.impl.store.counts.keys.IndexSampleKey;
-import org.neo4j.kernel.impl.store.counts.keys.IndexStatisticsKey;
-import org.neo4j.kernel.impl.transaction.CommittedTransactionRepresentation;
-import org.neo4j.kernel.impl.transaction.log.IOCursor;
-import org.neo4j.kernel.lifecycle.Lifespan;
-import org.neo4j.logging.NullLogProvider;
-import org.neo4j.kernel.monitoring.Monitors;
-
-import static org.neo4j.kernel.impl.pagecache.StandalonePageCacheFactory.createPageCache;
-
-public class RecountFromTransactions
-{
-    public static void main( String... args ) throws Exception
```

Comment:
Sorry, I didn't mean to remove tools and the likes. My bad. But yeah, we'll dig it out when we need it the next time around.